tools/windows_buildenv.bat

```diff
@@ -5,7 +5,8 @@ CALL :REALPATH "%~dp0\.."
 SET MIXXX_ROOT=%RETVAL%
 
 IF NOT DEFINED PLATFORM (
-    SET PLATFORM=x64
+    ECHO ^ERROR: The PLATFORM environment variable is not defined. Please ensure you are using the correct shell like 'x64 Native Tools Command Prompt for VS 20XX
+    EXIT /B 1
```

Comment:
```suggestion
if "%ARCH%"=="AMD64" (
    SET PLATFORM=x64
) else if "%ARCH%"=="ARM64" (
    SET PLATFORM=arm64
) else (
    echo Unknown processor architecture: %ARCH%
    PAUSE
    EXIT /B 1
)
```

Reply:
In the meantime I did a slight different approach.